src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar">
+      <div class="calendar-box calendar-box--1 start-day-wed"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--29 calendar--wed">
      <div class="calendar__day"></div>
```
Классы модификаторы должны быть у обертки ячеек, с их помощью устанавливается отступ слева для первой ячейки и определяется общее количество дней в месяце. 

У div, отвечающего за конкретный день - только один класс

Reply:
хотел спросить а каким образом модификатор без указаного дня должен определять каким днем недели будет первое число